applications_tests/lethe-fluid/heat_transfer_vof_lpbf_benchmark.mpirun=1.output

```diff
@@ -0,0 +1,199 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       16
+   Number of degrees of freedom: 75
+   Volume of triangulation:      0.36
+   Number of thermal degrees of freedom: 25
+   Number of VOF degrees of freedom: 25
+Initial refinement in box - Step  1 of 3
+   Number of active cells:       40
+   Number of degrees of freedom: 165
+   Volume of triangulation:      0.36
+   Number of thermal degrees of freedom: 55
+   Number of VOF degrees of freedom: 55
+Initial refinement in box - Step  2 of 3
+   Number of active cells:       88
+   Number of degrees of freedom: 339
+   Volume of triangulation:      0.36
+   Number of thermal degrees of freedom: 113
+   Number of VOF degrees of freedom: 113
+Initial refinement in box - Step  3 of 3
+   Number of active cells:       268
+   Number of degrees of freedom: 942
+   Volume of triangulation:      0.36
+   Number of thermal degrees of freedom: 314
+   Number of VOF degrees of freedom: 314
```

Comment:
This part should not be happening in a restart file. Maybe this option just has neverbeen tested with a restart. IF we are restarting, we should not do any mesh adaptation :)